frontend/src/scenes/onboarding/sdks/allSDKs.tsx

```diff
@@ -20,42 +20,79 @@ export const allSDKs: SDK[] = [
         docsLink: 'https://posthog.com/docs/libraries/js',
     },
     {
```

Comment:
I reorded this list to be alphabetical (except for web+snippet) so that it appears in order in the onboarding guide list